diff --git a/dist/types/eddsa-poseidon.d.ts b/dist/types/eddsa-poseidon.d.ts
index 28649a2370da06b7b570a855c66f92191fd58aa3..76713401eee6b9c5ce4549a509e356b234e55d86 100644
--- a/dist/types/eddsa-poseidon.d.ts
+++ b/dist/types/eddsa-poseidon.d.ts
@@ -2,7 +2,7 @@
 import { Point } from "@zk-kit/baby-jubjub";
 import type { BigNumberish } from "@zk-kit/utils";
 import { Buffer } from "buffer";
-import { Signature } from "./types";
+import { Signature } from "./types/index.js";
 /**
  * Derives a secret scalar from a given EdDSA private key.
  *
diff --git a/dist/types/index.d.ts b/dist/types/index.d.ts
index 779045dcbe9a66086cc778ba3f5027bd2e930f6d..30de9d02cc65990abb9afef63ddd19c429f5c88b 100644
--- a/dist/types/index.d.ts
+++ b/dist/types/index.d.ts
@@ -1,2 +1,3 @@
-export * from "./eddsa-poseidon";
-export * from "./types";
+export * from "./eddsa-poseidon.js";
+export * from "./types/index.js";
+
diff --git a/dist/types/utils.d.ts b/dist/types/utils.d.ts
index 19ad939a6a34734b875756a5eb48e876a231128d..551a5f925bfade764b962170b90e6f21b0d70e37 100644
--- a/dist/types/utils.d.ts
+++ b/dist/types/utils.d.ts
@@ -2,7 +2,7 @@
 import { Point } from "@zk-kit/baby-jubjub";
 import type { BigNumberish } from "@zk-kit/utils";
 import { Buffer } from "buffer";
-import { Signature } from "./types";
+import { Signature } from "./types/index.js";
 /**
  * Prunes a buffer to meet the specific requirements for using it as a private key
  * or part of a signature.
